data/mods/Aftershock/items/grenades.json

```diff
@@ -20,5 +20,113 @@
     "countdown_interval": 20,
     "countdown_action": { "type": "transform", "target": "null" },
     "flags": [ "TRADER_AVOID" ]
+  },
+  {
+    "id": "grenade_nuclear_detonator",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "cryo grenade" },
+    "description": "This is a military-grade cryo hand grenade.  Once it explodes it spreads exotic highly radioactive shrapnel.  Use this item to pull the pin and light the fuse.  You will then have five seconds before it explodes; throwing it would be a good idea.",
+    "weight": "397 g",
+    "volume": "250 ml",
+    "price": 1500,
+    "price_postapoc": 1000,
+    "to_hit": -1,
+    "bashing": 6,
+    "material": [ "steel", "plastic" ],
+    "symbol": "*",
+    "color": "green",
+    "use_action": {
+      "need_wielding": true,
+      "target": "grenade_nuclear_detonator",
+      "msg": "You pull the pin on the grenade.",
+      "target_charges": 5,
+      "active": true,
+      "menu_text": "Pull pin",
+      "type": "transform"
+    },
+    "flags": [ "LEAK_DAM", "RADIO_MODABLE", "RADIO_INVOKE_PROC", "BOMB", "GRENADE" ]
+  },
+  {
+    "id": "grenade_nuclear_detonator_act",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "active cryo grenade" },
+    "description": "This is an active grenade, and will explode any second now.  Once it explodes it spreads exotic highly thermodynamic shrapnel.  Better throw it!",
```

Comment:
I'll replace the first radioactive with thermodynamic to make it match since they are exotic materials that absorb heat  at an extremely fast rate.  Thanks for catching the mismatch :)

Reply:
Quite welcome; what is the grenade's ID from?